src/components/Cards/CashResult.vue

```diff
@@ -397,6 +397,7 @@ export default {
           financed_by: "altara",
           fixed_repayment: fixed_repayment,
           inventory_id: 2,
+          cost_price:this.downpaymentCalculations[0].cost_price,
```

Comment:
Is this cost price from the inventory itself?

Reply:
No, the costprice is whatever amount the users enters